Header header
uint32 id
float32 distortion
geometry_msgs/Pose pose
